documentation/Query-Syntax.md

```diff
@@ -127,13 +130,23 @@ Used to extract values from current state, for example, from `user`, `organisati
 Performs queries to a connected PostGres database and returns the result in a format specified by `type`.
 
 - Input:
-  - First child node contains a **string** representing the parameterized SQL query (i.e. $1, $2 substitution)
+  - First child node contains a **string** representing the parameterized SQL query (i.e. `$1`, `$2` substitution)
   - Remaining nodes return the values (**strings**, **numbers**, **arrays**) required for the above query substitution.
 - Output: Either **array** (all values flattened into one array), **string** (all results concatenated with spaces), or a single **number**, depending on the type specification. If not specified, it returns the default `node-postgres` format (an array of objects, with fields of each object being the database column names).
```

Comment:
Cool. Made a few changes, except:
- Didn't add a **Children** sub-heading as it would make it inconsistent with all the other operators.
- Not sure we need to say "node is an object", as I've said quite clearly in the intro that "Each node in the tree is an object.."
- Also, I think saying `{ value: **string** }` is misleading, as it could be another operator with its own children rather than a literal `value`. Key thing is that it _returns_ a string, so I've modified it to reflect that.